it/controllers/pensions/annualAllowance/AboveReducedAnnualAllowanceControllerISpec.scala

```diff
@@ -0,0 +1,333 @@
+  )
+
+  ".show" should {
+    userScenarios.foreach { user =>
+
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+        "render the above reduced annual allowance question page with no pre-filled radio buttons" which {
+          lazy val result: WSResponse = {
+            dropPensionsDB()
+            authoriseAgentOrIndividual(user.isAgent)
+            val pensionsViewModel = aPensionAnnualAllowanceViewModel.copy(reducedAnnualAllowanceQuestion = None)
+            insertCyaData(pensionsUserDataWithAnnualAllowances(pensionsViewModel), aUserRequest)
+            urlGet(fullUrl(aboveReducedAnnualAllowanceUrl(taxYearEOY)),
+              user.isWelsh, follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          radioButtonCheck(yesText, 1, checked = Some(false))
+          radioButtonCheck(noText, 2, checked = Some(false))
+          buttonCheck(buttonText, continueButtonSelector)
+          textOnPageCheck(expectedDetails, detailsSelector)
+          formPostLinkCheck(aboveReducedAnnualAllowanceUrl(taxYearEOY), formSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+
+        "render above reduced annual allowance question page with 'Yes' pre-filled when CYA data exists" which {
+          lazy val result: WSResponse = {
+            dropPensionsDB()
+            authoriseAgentOrIndividual(user.isAgent)
+            val pensionsViewModel = aPensionAnnualAllowanceViewModel.copy(reducedAnnualAllowanceQuestion = Some(true))
+            insertCyaData(pensionsUserDataWithAnnualAllowances(pensionsViewModel), aUserRequest)
+            urlGet(fullUrl(aboveReducedAnnualAllowanceUrl(taxYearEOY)), user.isWelsh, follow = false,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          radioButtonCheck(yesText, 1, checked = Some(true))
+          radioButtonCheck(noText, 2, checked = Some(false))
+          buttonCheck(buttonText, continueButtonSelector)
+          textOnPageCheck(expectedDetails, detailsSelector)
```

Comment:
Can you add a Link check too for the external link
Example [here](https://github.com/hmrc/income-tax-pensions-frontend/blob/main/it/controllers/pensions/annualAllowance/ReducedAnnualAllowanceTypeControllersControllerISpec.scala#L199)